BUILD.md

```diff
@@ -170,32 +170,43 @@ Until then, we advise Windows developers to use Visual Studio 2019.
 
 ## Prerequisites
 
-To build this package, you will need Python (>= 3.7),
-[Conan][] (>= 1.55), and [CMake][] (>= 3.16).
-
 > **Warning**
-> The commands in this document are not meant to be blindly copied and pasted.
+> The commands in this document are guidelines which cannot always be successfully copy and pasted.
 > This document is written for multiple audiences,
 > meaning that your particular circumstances may require some commands and not
 > others.
-> You should never run any commands without understanding what they do
-> and why you are running them.
+> It is recommended to fully understand what the following commands do before running them.
 >
 > These instructions assume a basic familiarity with Conan and CMake.
 > If you are unfamiliar with Conan,
 > then please read the [crash course](#a-crash-course-in-cmake-and-conan)
 > at the beginning of this document,
 > or the official [Getting Started][3] walkthrough.
 
+### All Platforms
+
+To build this package, you will need [Python][] (>= 3.7),
+[Conan][] (>= 1.52), and [CMake][] (>= 3.16).
+If you are unfamiliar with Conan,
+there is a crash course at the end of this document.
+
 [Conan]: https://conan.io/downloads.html
 [CMake]: https://cmake.org/download/
+[Python]: https://www.python.org/downloads/
+
+If you've never used Conan before, use autodetect to setup a default profile:
+
+```
+conan profile new default --detect
+```
```

Comment:
This command is already in the crash course. This section assumes the person has installed and configured Python, Conan, and CMake.

Reply:
Personally, I think it is good to repeat the command. Obviously, there is a balance (we wouldn't want to repeat it 5 times). But this looks ok to me.